framework/console/controllers/BaseMigrateController.php

```diff
@@ -41,6 +41,28 @@
      * or a file path.
      */
     public $templateFile;
+    /**
+     * @var array the template file for generating migration code automatically.
+     * This can be either a path alias (e.g. "@app/migrations/template.php")
+     * or a file path.
+     */
+    public $templateFileGenerators;
+    /**
+     * @var array the fields
+     */
+    public $fields;
+    /**
+     * @var string the primary key name
+     */
+    public $primaryKey = 'id';
+    /**
+     * @var string
+     */
+    public $createAt = 'create_at';
```

Comment:
if user not specified timestamps and primaryKey it is generate automatically, for example : 

```
// user not specified timestamps and primaryKey
yii migrate/create CreatePost --fields=body:text 
```
Generate : 

```php
$this->createTable('post', [
  'id' => $this->primaryKey(),
  'body' => $this->text(),
  'create_at' => $this->integer(),
  'update_at' => $this->integer()
]);
```

```
// user specified timestamps and primaryKey
yii migrate/create CreatePost --fields=body:text --primaryKey=title --createAt=create --updateAt=update
```

Generate : 

```php
$this->createTable('post', [
  'title' => $this->primaryKey(),
  'body' => $this->text(),
  'create' => $this->integer(),
  'update' => $this->integer()
]);
```